src/DetailsView/components/settings-panel/settings-panel.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Name seems redundant if it's under 'settings-panel' folder. Maybe folder name should be 'settings'?

Reply:
I get your point, but there is a settings folder inside settings-panel... I would prefer to rename the class. Panel sounds too generic though. Any other ideas?